src/open_inwoner/utils/logentry.py

```diff
@@ -28,7 +28,7 @@ def get_change_message(fields=None, form=None):
         form, form.changed_data
     )
 
-    return _("Changed {changed_fields}.").format(
+    return _("Changed: {changed_fields}.").format(
```

Comment:
I don't know if it's really necessary, but we could add here a different message when the user doesn't make an actual change to any field (like django admin's message).

![actions](https://user-images.githubusercontent.com/63058071/172854993-bb71dd26-1f49-46b5-af6d-f727e324eecd.png)

![admin](https://user-images.githubusercontent.com/63058071/172855034-98e873f7-ae5d-4cd2-ab5e-3ace84cbe423.png)

Reply:
Thanks for catching it! I think there is no point in logging form submission without the actual change, so I excluded such cases from logging. Now it should be fixed.